Godeps/Godeps.json

```diff
@@ -143,18 +151,18 @@
 		},
 		{
 			"ImportPath": "github.com/google/cadvisor/client",
-			"Comment": "0.5.0-76-gd74485c",
-			"Rev": "d74485ce77d81101c059efc0841deb44ae177258"
+			"Comment": "0.5.0-165-g8064a13",
```

Comment:
Update deps. I don't think the version number here matches the cadvisor's versioning system.